src/style.css

```diff
@@ -0,0 +1,124 @@
+/* #region General styles + Stars margin */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+/* #endregion General styles + Stars margin */
+
+/* #region Non-active stars styles */
+
+.stars--0 > .stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--1 > .stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--2 > .stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--3 > .stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--4 > .stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--5 > .stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+/* #endregion Non-active stars styles */
+
+/* #region Active stars styles */
+
+.stars--1 > .stars__star:nth-child(-n+1) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--2 > .stars__star:nth-child(-n+2) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--3 > .stars__star:nth-child(-n+3) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
+
+.stars--5 > .stars__star:nth-child(-n+5) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  /* background-size: contain; */
+  background-repeat: no-repeat;
+}
```

Comment:
How do you think it is okay to duplicate many such styles?  [Checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md) 2nd point